packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer-card.tsx

```diff
@@ -77,7 +102,7 @@ const ContractDrawerCard = observer(
                 contract_info={contract_info}
                 contract_update={contract_update}
                 currency={currency}
-                current_focus={current_focus}
+                current_focus={current_focus ?? ''}
```

Comment:
What is considered to be better approach in this case: to add default value to prop or to pass an empty string?